src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,37 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Settings.Constants;
+
+public class Climber extends SubsystemBase {
+  TalonSRX lift;
+
+  /**
+   * Creates a new Climber.
+   */
+  public Climber() {
+    lift = new TalonSRX(Constants.Climber.liftMotorID);
+
```

Comment:
make sure to set talon to factory default to make sure no funny business occurred.